src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+
+{{ figure_markup(
+  image="alt-attribute-lengths.png",
+  caption="`alt` attribute lengths.",
+  description="A bar chart showing the distribution of `alt` attribute lengths for images on desktop pages in 2025. About 13% of images have no `alt` attribute and 30% use an empty `alt`. Of the remaining images with non-empty text, most are short: around 20% have 10 or fewer characters, 17% have 20 or fewer, 18% have 30 or fewer, and only about 1% have up to 100 characters. Virtually none exceed 100 characters.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQFD-7C6Jv6q1JyviDsKosRlVwaok7g7nRCQ9NGMw5MaAAohL7EcDejVwgp13Z_T2S_57Zi0YaVb7st/pubchart?oid=536884664&format=interactive",
+  sheets_gid="785975896",
+  sql_file="common_alt_text_length"
+  )
+}}
+
+Best practices continue to emphasize providing concise yet descriptive alt text tailored to image context, avoiding filenames, and using semantic file formats like SVG when appropriate. Artificial Intelligence (AI) tools show promise too. <a hreflang="en" href="https://www.drupal.org/project/ai_image_alt_text">Drupal's integration of AI-assisted alt text suggestions</a> helps authors create better alt attributes by providing editable examples. Brian Teeman wrote an interesting critique of the <a hreflang="en" href="https://magazine.joomla.org/all-issues/june-2024/ai-generated-alt-text">AI generation of Alt Text</a>.
+
+Images remain an area with opportunities for significant accessibility improvement despite steady progress.
+
+### Audio and video
+
+The HTML `<track>` element provides timed text tracks like captions, subtitles, and audio descriptions for media elements like `<video>` and `<audio>`. In 2025, this element is still underutilized. Despite its importance for users who are deaf, hard of hearing, or blind, adoption rates stay exceptionally low, at under 1%.
+
+Many modern video platforms now commonly use [HTTP Live Streaming (HLS)](https://en.wikipedia.org/wiki/HTTP_Live_Streaming) instead of the native `<track>` element. This may contribute to the low usage statistics. HLS, which requires custom implementation, means that developers need to be more intentional about building in caption support themselves. This means there's more room for developers to forget about captions or do them poorly.
+
+Captions are essential for deaf and hard of hearing users. They also benefit viewers in noisy environments or those with difficulty understanding spoken language. Audio descriptions enable users with visual impairments to gain context about visual content.
+
+Compared to 2024, we've seen no significant growth in `<track>` usage for captions and subtitles, indicating that the industry still has substantial room for improvement. This is particularly true for media embedded through third-party services or `<iframe>` elements, which are less likely to offer accessible alternatives directly.
```

Comment:
> This is particularly true for media embedded through third-party services or `<iframe>` elements, which are less likely to offer accessible alternatives directly.

Is that true? YouTube includes captions (and auto captions - which while of questionable quality sometimes, are better than nothing IMHO!).

Reply:
I think that’s the same as what you already have.

I can’t honestly say whether it’s more or less likely to be accessible when it’s a third party video service. Gut feel is YouTube is slightly more accessible in terms of subtitles for the reasons I said, but that’s more a personal opinion (and I’m probably biased, since I work for Google, even if not on YouTube) and I dunno if they have other accessibility issues that counteract that gain. Was just something I thought worth mentioning in case you hadn’t considered that.

So, happy to leave it as is and just resolve this comment.

Reply:
I've removed that line altogether :)